libmscore/textbase.cpp

```diff
@@ -2574,10 +2576,13 @@ void TextBase::draw(QPainter* p) const
             if (circle())
                   p->drawEllipse(frame);
             else {
-                  int r2 = frameRound();
+                  qreal baseSpatium = MScore::baseStyle().value(Sid::spatium).toDouble();
+                  qreal frameRoundFactor = ((sizeIsSpatiumDependent() ? spatium() : baseSpatium)/baseSpatium) /2;
```

Comment:
Looks overcomplicated. `frameRoundFactor` could be equal to `spatium() / (baseSpatium / 2)` or `1/2`?
Let's write so: 
```
qreal frameRoundFactor = (sizeIsSpatiumDependent() ? spatium() / (baseSpatium / 2) : 0.5f;
```

Reply:
`frameRoundFactor` could be equal to '(spatium()/baseSpatium) / 2` or `1/2`, which I guess is the same as what you wrote. Fixed.